nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -202,8 +202,18 @@ double VelocitySmoother::applyConstraints(
   const double accel, const double decel, const double eta)
 {
   double dv = v_cmd - v_curr;
-  const double v_component_max = accel / smoothing_frequency_;
-  const double v_component_min = decel / smoothing_frequency_;
+
+  double v_component_max;
+  double v_component_min;
+
+  if (abs(v_cmd) >= abs(v_curr)) {
```

Comment:
What will be if `v_curr = 1.0` while `v_cmd = -1.2`? I think the comparison should also count the sign of current and required velocities. In the current example, we have the deceleration and then acceleration as depicted at the graph below:
![20230410_130139](https://user-images.githubusercontent.com/60094858/230880538-417944ca-e400-4313-b4aa-37871cf8e175.jpg)